src/dra/dra.h

```diff
@@ -411,9 +411,11 @@ extern AnimationFrame D_800ADC10[];
 extern AnimationFrame D_800ADC44[];
 extern u16 D_800ADC7C[2];
 extern Point16* D_800ADCC8[];
+extern u8 D_800AE120[];
 extern RECT D_800AE130;
 extern s32 D_800AE270[9];
 extern AnimationFrame D_800AE294[];
+extern Unkstruct_800ADEF0 D_800ADEF0[];
```

Comment:
Storing this in a shared header is breaking the build. For simplicity I suggest to just move it into `7E4BC.c`